### DISCUSSION TITLE
build on macOS

### DISCUSSION THREAD

**Discussion by krishpranav** [ravynsoft/ravynos#96] [Q&A] [answered]
hi @airyx team so I wanted to build airyx on my Mac machine can you provide the build instructions

**Comment by mszoek** [comment, marked as answer]
Hi! Airyx will only currently build on itself (it's self-hosting) or on vanilla FreeBSD 12.2 in multiple stages. I develop it on a MacBook using an [Airyx 0.3](https://github.com/mszoek/airyx/releases/) VM running in Vmware Fusion. That's the easiest way. If the ISO image boots on your hardware and you don't mind overwriting macOS (or have a spare drive), you could install it natively.

**Reply by krishpranav** [reply]
thank you @mszoek
